src/create_disk.sh

```diff
@@ -31,6 +31,7 @@ set -x
 
 # partition and create fs
 sgdisk -Z $disk \
+	-U 00000000-0000-0000-0000-000000000001 \
```

Comment:
This one's worth a comment in the source code too; commit message is good but code motion can make it harder to find.

Also maybe better to use a real (hardcoded) UUID instead, to avoid anyone else possibly ascribing special meaning to this `1` GUID?

Reply:
+1 to adding a comment.

I mean this `1` guid is special, we're using it as a sentinel. It however is also not a valid uuid since it's variant and version is unset. What about `00000000-0000-1000-a000-000000000001` (version 4 (random) variant 1 [1]). We didn't hit any trouble with CL with the invalid `1` guid, but when testing locally a `0` guid seemed to cause problems, so having something valid is probably good. I'd like to keep it as something that sticks out as an uninitialized value.

[1] https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)